homeassistant/components/braviatv/button.py

```diff
@@ -53,8 +53,7 @@ async def async_setup_entry(
     assert unique_id is not None
 
     async_add_entities(
-        BraviaTVButton(coordinator, unique_id, config_entry.title, description)
```

Comment:
while we did store the model in the entry title, users can change it, which is weird